src/robusta/core/reporting/base.py

```diff
@@ -187,11 +189,32 @@ def attribute_map(self) -> Dict[str, str]:
             "name": str(self.subject.name),
         }
 
+    def _map_service_to_uri(self):
+        if not self.service:
+            return "graphs"
+        if self.service.resource_type.lower() == "job":
+            return "jobs"
+        return "services"
+
+    def get_investigate_uri(self, account_id: str, cluster_name: Optional[str] = None):
+        uri_path = self._map_service_to_uri()
+        params = {
+            "account": account_id,
+            "clusters": f"[\"{cluster_name}\"]" if cluster_name else None,
+            "namespaces": f"[\"{self.subject.namespace}\"]" if self.subject.namespace else None,
+            "kind": self.service.resource_type if self.service else None,
+            "name": self.service.name if self.service else None,
```

Comment:
I think this might be confusing
`name` is ok I think
but `names` is a little confusing.
can we call it `key` or something?

Reply:
The same thing - these are the parameters in the query string

Reply:
got it